cookbooks/aws-parallelcluster-platform/templates/nvidia-imex/nvidia-imex.service.erb

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=NVIDIA IMEX service
+After=network-online.target
+Requires=network-online.target
+
+[Service]
+Environment="KRB5_CLIENT_KTNAME=/etc/krb5.keytab"
```

Comment:
Why do we need kerberos configuration?
What if the user has configured kerberos to point to a different principal file overriding `default_keytab_name` in `/etc/krb5.conf`?